Updates helm values for consul-k8s 1.0.9 (#18447)
hashicorp/consul: website/content/docs/k8s/helm.mdx

    - `secretKey` ((#v-global-gossipencryption-secretkey)) (`string: ""`) - The key within the Kubernetes secret or Vault secret key that holds the gossip
      encryption key.

>     - `logLevel` ((#v-global-gossipencryption-loglevel)) (`string: ""`) - Override global log verbosity level for gossip-encryption-autogenerate-job pods. One of "trace", "debug", "info", "warn", or "error".

Reviewer: ```suggestion
    - `logLevel` ((#v-global-gossipencryption-loglevel)) (`string: ""`) - Override global log verbosity level for `gossip-encryption-autogenerate-job` pods. One of "trace", "debug", "info", "warn", or "error".
```

      - `secretKey` ((#v-global-tls-cakey-secretkey)) (`string: null`) - The key within the Kubernetes or Vault secret that holds the CA key.

>     - `annotations` ((#v-global-tls-annotations)) (`string: null`) - This value defines additional annotations for
>       tls init jobs. This should be formatted as a multi-line string.

Reviewer: ```suggestion
    - `annotations` ((#v-global-tls-annotations)) (`string: null`) - This value defines additional annotations for
      tls init jobs. Format this value as a multi-line string.
```

        beta.kubernetes.io/arch: amd64
      ```

>     - `annotations` ((#v-global-acls-annotations)) (`string: null`) - This value defines additional annotations for
>       acl init jobs. This should be formatted as a multi-line string.

Reviewer: active voice

```suggestion
    - `annotations` ((#v-global-acls-annotations)) (`string: null`) - This value defines additional annotations for
      acl init jobs. Format this value as a multi-line string.
```

        -o jsonpath="{.clusters[?(@.name=='<your cluster name>')].cluster.server}"
      ```

>     - `logLevel` ((#v-global-federation-loglevel)) (`string: ""`) - Override global log verbosity level for the create-federation-secret-job pods. One of "trace", "debug", "info", "warn", or "error".

Reviewer: ```suggestion
    - `logLevel` ((#v-global-federation-loglevel)) (`string: ""`) - Override global log verbosity level for the `create-federation-secret-job` pods. One of "trace", "debug", "info", "warn", or "error".
```

- `meshGateway` ((#v-meshgateway)) - [Mesh Gateways](https://developer.hashicorp.com/consul/docs/connect/gateways/mesh-gateway) enable Consul Connect to work across Consul datacenters.

  - `enabled` ((#v-meshgateway-enabled)) (`boolean: false`) - If [mesh gateways](https://developer.hashicorp.com/consul/docs/connect/gateways/mesh-gateway) are enabled, a Deployment will be created that runs
    gateways and Consul service mesh will be configured to use gateways.
    This setting is required for [Cluster Peering](https://developer.hashicorp.com/consul/docs/connect/cluster-peering/k8s).
    Requirements: consul 1.6.0+ if using `global.acls.manageSystemACLs``.

>   - `logLevel` ((#v-meshgateway-loglevel)) (`string: ""`) - Override global log verbosity level for mesh-gateway-deployment pods. One of "trace", "debug", "info", "warn", or "error".

Reviewer: ```suggestion
  - `logLevel` ((#v-meshgateway-loglevel)) (`string: ""`) - Override global log verbosity level for `mesh-gateway-deployment` pods. One of "trace", "debug", "info", "warn", or "error".
```

  - `enabled` ((#v-ingressgateways-enabled)) (`boolean: false`) - Enable ingress gateway deployment. Requires `connectInject.enabled=true`
    and `client.enabled=true`.

>   - `logLevel` ((#v-ingressgateways-loglevel)) (`string: ""`) - Override global log verbosity level for ingress-gateways-deployment pods. One of "trace", "debug", "info", "warn", or "error".

Reviewer: ```suggestion
  - `logLevel` ((#v-ingressgateways-loglevel)) (`string: ""`) - Override global log verbosity level for `ingress-gateways-deployment` pods. One of "trace", "debug", "info", "warn", or "error".
```
